### PR TITLE
Add support for Soyuz numpad

### PR DESCRIPTION
**Pull request by ai03-2725**
## Description

Adds support for the Soyuz numpad.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Adds support for the [Soyuz numpad kit](https://github.com/ai03-2725/Soyuz)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ai03/soyuz/rules.mk [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# MCU name
+#MCU = at90usb1286
```
Comment (e11i0t23) [inline keyboards/ai03/soyuz/rules.mk]:
```suggestion
```

File: keyboards/ai03/soyuz/config.h [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+//   #define RGBLIGHT_EFFECT_SNAKE
+//   #define RGBLIGHT_EFFECT_KNIGHT
+//   #define RGBLIGHT_EFFECT_CHRISTMAS
+//   #define RGBLIGHT_EFFECT_STATIC_GRADIENT
+//   #define RGBLIGHT_EFFECT_RGB_TEST
+//   #define RGBLIGHT_EFFECT_ALTERNATING
+// #endif
+
+/* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
+#define DEBOUNCING_DELAY 5
```
Comment (e11i0t23) [inline keyboards/ai03/soyuz/config.h]: This has since been renamed
```suggestion
#define DEBOUNCE 5
```

File: keyboards/ai03/soyuz/soyuz.h [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+/* This a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+ 
+#define LAYOUT( \
```
Comment (drashna) [inline keyboards/ai03/soyuz/soyuz.h]:
```suggestion
#define LAYOUT_ortho_5x4( \
```

File: keyboards/ai03/soyuz/rules.mk [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support (+2400 to 4200, depending on config)
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs (+400)
```
Comment (drashna) [inline keyboards/ai03/soyuz/rules.mk]:
```suggestion
HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs (+400)

LAYOUTS = ortho_5x4
```

File: keyboards/ai03/soyuz/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT( /* Base */
```
Comment (drashna) [inline keyboards/ai03/soyuz/keymaps/default/keymap.c]:
```suggestion
  [0] = LAYOUT_ortho_5x4( /* Base */
```

File: keyboards/ai03/soyuz/keymaps/1U/keymap.c [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+ */
+#include QMK_KEYBOARD_H
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+  DBLZERO = SAFE_RANGE
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT( /* Base */
```
Comment (drashna) [inline keyboards/ai03/soyuz/keymaps/1U/keymap.c]:
```suggestion
  [0] = LAYOUT_ortho_5x4( /* Base */
```

File: keyboards/ai03/soyuz/info.json [hunk 7]
```diff
@@ -0,0 +1,33 @@
+{
+    "keyboard_name": "Soyuz", 
+    "url": "https://github.com/ai03-2725/soyuz", 
+    "maintainer": "ai03", 
+    "width": 4, 
+    "height": 5, 
+    "layouts": {
+        "LAYOUT": {
```
Comment (drashna) [inline keyboards/ai03/soyuz/info.json]:
```suggestion
        "LAYOUT_ortho_5x4": {
```

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**